doc/diagram.dot

```diff
@@ -0,0 +1,25 @@
+digraph desdemona {
+        onyx [label="Onyx cluster"];
+
+        subgraph frontend {
+                dashboard [label="Dashboard"];                
+        }       
+
+        dashboard -> onyx [label="Queries"];
+        onyx -> dashboard [label="Live data"];
```

Comment:
Since we have live data showing, should there also be something that shows queries can go to the local cache?